wagtail_ab_testing/models.py

```diff
@@ -77,7 +73,7 @@ class AbTest(models.Model):
     page = models.ForeignKey('wagtailcore.Page', on_delete=models.CASCADE, related_name='ab_tests')
     name = models.CharField(max_length=255)
     hypothesis = models.TextField(blank=True)
-    variant_revision = models.ForeignKey(get_revision_model(), on_delete=models.CASCADE, related_name='+')
+    variant_revision = models.ForeignKey('wagtailcore.Revision', on_delete=models.CASCADE, related_name='+')
```

Comment:
Actually there was more that could be done here + a new migration.

This should fix sites that have the package already installed but the `treatment_revision` foreign key needs to be re-mapped as well.

Reply:
I'm having second thoughts about this change. It probably needs more work to update the variant_revision fields. Like `rebuild_references_index` command in wagtail.

Reply:
I agree with you @nickmoreton that this is not enough to migrate to the `Revision` model. I'm investigating the best way to go about this.

Reply:
I've implemented a fix for this in #52 by squashing migrations. See the explanation in the PR description.